specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPacks_API.json

```diff
@@ -71,7 +71,7 @@
         ]
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Insights/queryPacks": {
```

Comment:
>Insights [](start = 57, length = 8)

You are removing resource types from a preview API version that is older than a year. This is considered a breaking change and will require approval. Can you explain?

Breaking changes require a separate approval. See instructions at [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning).